README.md

```diff
@@ -15,6 +15,27 @@ with any kind of software without having to release its source code.
 Additionally, the example code has been released to the public domain (see the
 [separate license](examples/COPYING) for more information).
 
+Quick Started
+===============
+
+**Prerequisites** for building libgit2:
+
+1. [CMake](https://cmake.org/)
+2. [Python](https://www.python.org) is used by our test framework.
+3. C compiler
+   * Windows: Visual Studio is recommended
+   * Mac: Xcode is recommended
+
+**Build**
+
+1. Create a build directory beneath the libgit2 source directory, and change
+   into it: `mkdir build && cd build`
+2. Create the cmake build environment: `cmake ..`
+3. Build libgit2: `cmake --build .`
```

Comment:
These instructions may work just fine for users used to the command line. But maybe we should also add instructions for the CMake GUI?